src/styles/bar.css

```diff
@@ -5,7 +5,22 @@ body {
   height: 100vh;
 }
 
+
 [role=application] {
   height: initial;
   flex: 1 100vh;
 }
+
+
+[role=banner] {
+  --silver: #d6d8da;
```

Comment:
@m4dz Are you sure autoprefixer supports custom properties? I tried using CSSNext syntax on a project once and I had to use the postcss plugin **cssnext** to make it work. 
CSSNext plugin brings autoprefixer with it so i'm not sure which does what but here's what I know. 👅